### DISCUSSION TITLE
WSA apps are locked on sleep screen with the clock

### DISCUSSION THREAD

**Discussion by keyven** [LSPosed/MagiskOnWSALocal#303] [Q&A] [answered]
Hello since few days I cannot use anymore the WSA. 

I have tried to update to the last build but I am still stuck. 

Notifications are working fine 
![image](https://user-images.githubusercontent.com/11938471/208493014-d9c1b324-ae31-4f69-b9e5-1d05c54062d2.png)

But when I launch an app I always have a full screen with the clock and nothing available. I cannot use the mouse or the keyboard; I am able to connect with ADB but no command wake up the WSA. 

![image](https://user-images.githubusercontent.com/11938471/208492788-10855350-21b3-4d32-861a-388e2dd3dd01.png)

I am on windows 11 beta ring
![image](https://user-images.githubusercontent.com/11938471/208492982-6e825f38-41a8-4cf0-8f82-f0f125f8b6fe.png)

Any idea or help is welcome. 
Best regards,

**Comment by Howard20181** [comment, marked as answer]
https://github.com/LSPosed/MagiskOnWSALocal/discussions/172#discussioncomment-3830977

**Reply by keyven** [reply]
Thanks a lot for the answer. 

I had to uninstall and reinstall the WSA, just a reset was not enough (it took a few second but you loose everything).
If it can help anyone, don't install apps that requiere advance security (in my case it was a password manager). 

Best regards,

**Reply by Howard20181** [reply]
For wipe data you need to delete `userdata.vhdx` manually rather than click reset in WSA settings.
